src/pixie/images.nim

```diff
@@ -406,17 +402,24 @@ proc drawCorrect*(a, b: Image, mat: Mat3, blendMode: BlendMode) =
     minFilterBy2 /= 2
     matInv = matInv * scale(vec2(0.5, 0.5))
 
+  let smooth = not(dx.length == 1.0 and dy.length == 1.0 and
```

Comment:
I would like to keep draw correct free of "fancy" logic. 

Could you please create a different version of draw correct with the vector motion if you want to test it, but keep draw correct as simple and plain as possible?

Reply:
I removed it, not important to me was just comparing